app/code/core/Mage/GoogleAnalytics/Block/Ga.php

```diff
@@ -329,45 +337,79 @@ protected function _getOrdersTrackingCodeAnalytics4()
             $productCollection = Mage::getSingleton('checkout/session')->getQuote()->getAllVisibleItems();
             $eventData = [];
             $eventData['currency'] = Mage::app()->getStore()->getCurrentCurrencyCode();
-            $eventData['value'] = 0.00;
+            $eventData['value'] = '0.00';
```

Comment:
```suggestion
            $eventData['value'] = 0.00;
```

Shouldn't we leave this as is (float), so that ```$eventData['value'] += $_product->getFinalPrice();``` isn't a string being added to a float?

it's formatted as string properly here ```$eventData['value'] = number_format($eventData['value'], 2);``` before being output.